packages/compass-e2e-tests/tests/search-indexes.test.ts

```diff
@@ -203,6 +203,8 @@ describe('Search Indexes', function () {
   });
 
   afterEach(async function () {
+    await screenshotIfFailed(compass, this.currentTest);
```

Comment:
Drive-by: One of the new search indexes tests sometimes flakes and I noticed that it disconnected BEFORE we take the error screenshot and then obviously the screenshot is useless. This is something I broke probably in the previous e2e PR and just didn't notice.